taxi/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.db.models.constraints import UniqueConstraint
+
+
+class Manufacturer(models.Model):
+    name = models.TextField()

```

Comment:
I don’t know if this is correct or not, but usually, we wrote CharField() for such fields, and TextField is more for description.

But better write:
`name = models.CharField(max_length=255)`